TAInformation/Models/instructor.py

```diff
@@ -1,21 +1,53 @@
 from TAInformation.Models.account_type import AccountType
 from TAInformation.Models.base_user import BaseUser
+from TAInformation.Models.validator_methods import *
+from TAInformation.models import Course, User, LabCourseJunction, Lab
 
 
 class Instructor(BaseUser):
     # Constructor
     def __init__(self, id_number: int, name: str, password: str, email: str, address: str, phone: str):
         super().__init__(id_number, name, password, email, address, phone)
         self.role = AccountType.INSTRUCTOR
-
+    # precondition: none
+    # post condition: return an array of all Course assignments for instructor
     def display_courses(self):
```

Comment:
Overall, LGTM! These changes to display_courses might break the styling and formatting of courses, but we will look at that during today's meeting.

Reply:
i've tested them out, they look the same as the admin, no empty spaces or empty bullet pts.